### PR TITLE
refactor(showcase): add missing / improve showcase examples for business

### PR DESCRIPTION
**Pull request by vlanz**
add missing showcases and simplify existing showcase templates for business

### CODE REVIEW

File: src/showcase/app/business/business-examples/accordion-examples/basic-accordion-example/basic-accordion-example.component.ts [hunk 1]
```diff
@@ -0,0 +1,103 @@
+import { Component, OnDestroy, OnInit, ViewChild } from '@angular/core';
```
Comment (jeripeierSBB) [inline src/showcase/app/business/business-examples/accordion-examples/basic-accordion-example/basic-accordion-example.component.ts:1]: To reduce complexity i'd suggest to only use reactiveForms for disabled, panelMode and hideToggle. Can also be done later...

File: src/showcase/app/business/business-examples/accordion-examples/basic-accordion-example/basic-accordion-example.component.ts [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+    },
+  ];
+
+  constructor() {
+    this.accordionForm = new FormGroup({
+      radioModes: new FormControl({ value: null, disabled: true }),
+      disabled: new FormControl(this.disabled),
+      multi: new FormControl(this.multi),
+      panelMode: new FormControl(this.panelMode),
+      showToggle: new FormControl(this.hideToggle),
```
Comment (jeripeierSBB) [inline src/showcase/app/business/business-examples/accordion-examples/basic-accordion-example/basic-accordion-example.component.ts:45]:
```suggestion
      hideToggle: new FormControl(this.hideToggle),
```
otherwise there are console errors

File: src/showcase/app/business/business-examples/accordion-examples/wizard-accordion-example/wizard-accordion-example.component.ts [hunk 3]
```diff
@@ -0,0 +1,120 @@
+import { Component, OnDestroy, OnInit, ViewChild } from '@angular/core';
```
Comment (jeripeierSBB) [inline src/showcase/app/business/business-examples/accordion-examples/wizard-accordion-example/wizard-accordion-example.component.ts:1]: Please remove all non content related code.

**Comment by vlanz** [comment]
This PR should be ready for review again!

File: src/showcase/app/public/BUILD.bazel [hunk 4]
```diff
@@ -28,7 +28,6 @@ ng_ts_library(
         "//src/showcase/app/public/public-examples/button-examples",
         "//src/showcase/app/public/public-examples/captcha-examples",
         "//src/showcase/app/public/public-examples/checkbox-examples",
-        "//src/showcase/app/public/public-examples/checkbox-panel-examples",
```
Comment (kyubisation) [inline src/showcase/app/public/BUILD.bazel:31]: Why was this removed?

File: src/showcase/app/public/public-examples/field-examples/field-examples.module.ts [hunk 5]
```diff
@@ -1,11 +1,13 @@
...
-import { DatepickerModule } from '@sbb-esta/angular-public/datepicker';
-import { FieldModule } from '@sbb-esta/angular-public/field';
-import { SelectModule } from '@sbb-esta/angular-public/select';
+import {
+  ButtonModule,
+  CheckboxModule,
+  DatepickerModule,
+  FieldModule,
+  SelectModule,
+} from '@sbb-esta/angular-public';
```
Comment (kyubisation) [inline src/showcase/app/public/public-examples/field-examples/field-examples.module.ts:10]: Don't import from base import. Prefer secondary entrypoints.

File: src/angular-core/scrolling/BUILD.bazel [hunk 6]
```diff
@@ -41,8 +41,8 @@ ng_test_library(
 ng_web_test_suite(
     name = "unit_tests",
     srcs = [
-        "@npm//:node_modules/resize-observer-polyfill/dist/ResizeObserver.js",
         "@npm//:node_modules/perfect-scrollbar/dist/perfect-scrollbar.js",
+        "@npm//:node_modules/resize-observer-polyfill/dist/ResizeObserver.js",
```
Comment (kyubisation) [inline src/angular-core/scrolling/BUILD.bazel:45]: This file should not be changed.
I will fix this the schematics in the future, but for now don't commit changes to this file.

File: src/showcase/app/public/BUILD.bazel (hunk 4 above)
Comment (vlanz) [reply src/showcase/app/public/BUILD.bazel]: My bad, since it wasn't an example present in the business package I must've overwritten it by mistake... ⚰️

**Comment by kyubisation** [comment]
@vlanz Thank you very much for your work! 😃
